airbyte-api/src/main/openapi/config.yaml

```diff
@@ -2953,6 +2999,28 @@ components:
     CompleteOauthResponse:
       type: object
       additionalProperties: true # Oauth parameters like refresh/access token etc.. will be different per API so we don't specify them in advance
+    SetInstancewideSourceOauthParamsRequestBody:
+      type: object
+      required:
+        - sourceConnectorDefinitionId
+        - params
+      properties:
+        sourceDefinitionId:
+          $ref: "#/components/schemas/SourceDefinitionId"
+        params:
+          type: object
+          additionalProperties: true
```

Comment:
If we also add `workspaceId` as an optional properties (not required), we can also handle setting workspace specific parameters.

Thus, I wouldn't include the `instanceWide` in the naming

Reply:
I am intentionally including this so that endpoint permissioning is possible without having to inspect the body of the request i.e: I want to make this whole endpoint available only to super admins rather than depending on whether workspace ID is passed, make it available to different permission levels. WDYT?

Reply:
Ok that makes sense!